packages/common-utils/src/clickhouse/index.ts

```diff
@@ -431,13 +433,17 @@ export abstract class BaseClickhouseClient {
     password,
     queryTimeout,
     application,
+    requestTimeout,
```

Comment:
Can we remove the `// TODO: make configurable` from line 428 now?

Reply:
Yup. Done.